ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/CPAN/tasks/main.yml

```diff
@@ -32,6 +32,8 @@
 # All
 - name: Install JSON
   command: cpanm --with-recommends --sudo JSON
+  when:
+    - not ((ansible_distribution == "RedHat" or ansible_distribution == "CentOS") and ansible_distribution_major_version == "8") 
```

Comment:
@smlambert Am I correct in saying that we no longer need these cpan packages at all? If so we should just remove this role.